### ISSUE TITLE
The Fingerprinters resources are "Allowed" even if they are blocked under the "Cross-Site Tracking Cookies" while enrolled in the treatmentScrip branch

### ISSUE THREAD

**Issue by SoftVision-CosminMuntean** [mozilla/fingerprinting-blocking-methods-study-addon#12]
[Affected Platforms]:
- All Windows
- All Mac
- All Linux

[Affected Versions]:
- Firefox 69.0 Unbranded build
- Firefox 70.0.1 Unbranded build
- Firefox 71 Unbranded build

[Prerequisites]:
- Have a Firefox 69, 70 or 71 Unbranded build. 
- Have access to the stage delivery console.
- Have an active branched-addon recipe  for the Fingerprinting Implementation Retention Study V1. You can use my created recipe.
- Have a user.js file that contains all the prefs needed to be targeted by the Normandy recipe. You can use my created user.js.

[Steps to reproduce]:
1. Open the browser with the profile from prerequisites.
2. In the profile directory copy the user.js file.
3. Restart the browser.
4. Navigate to a website that contains Fingerprinting tracks eg: https://www.skyscanner.nl/
5. Click the "Shield" icon from the URL bar.
6. Observe the "Content Blocking" section from the panel.
7. Click the "Fingerprinters" and observe the resource.

[Expected result]:
Step 6: The "Fingerprinters" are displayed under the "Blocked" section.
Step 7: The specific trackers are blocked.

[Actual result]:
Step 6: The "Fingerprinters" are displayed under the "Allow" section.
Step 7: The specific trackers are not blocked. However, it seems that are blocked under the "Cross-Site Tracking Cookies"

[Notes]:
- Here is a link with the screen recording of the issue: [link](https://drive.google.com/file/d/1sXbmlu9yr27iTU3-rhPmbMQsgkgw6-an/view?usp=sharing).

**Comment by birdsarah** [comment]
Again @SoftVision-CosminMuntean, thank you for your detailed write-ups and diligence. My response here is effectively a copy-paste from #11, but I am doing so, in case the conversation continues, we can keep the conversation separate on each issue.

TLDR; Everything is working as expected. 

In standard ETP, fingerprinting blocking happens at the domain level. The door hanger is describing the activities of ETP in allowing or blocking whole domains. 

In the treatmentScript branch, we are not doing whole domain blocking. Therefore the fingerprinting domains are allowed. Hence the UI. However, in the treatmentScript branch we also expect to see, in the browser console, in normal browsing only, a note that a specific script was blocked.

I will agree this is a confusing interaction with the Firefox UI. The door hanger says the fingerprinting domain perimetrix.net was allowed, but we have, behind the scenes blocked a specific fingerprinting script from this domain.

This lack of fine grained UI is a known limitation of the study, and we believe it's acceptable.

ping @mlopatka - we have already discussed this, but as @SoftVision-CosminMuntean brought it up, just wanted to check that you are still on board with this approach. If yes, then we can close this issue.

**Comment by mlopatka** [comment]
@SoftVision-CosminMuntean @birdsarah I am still on board with this approach.

**Comment by SoftVision-CosminMuntean** [comment]
@birdsarah @mlopatka thank you for looking over the issues and for clarification.
